aspnetcore/signalr/javascript-client/sample/wwwroot/js/chat.js

```diff
@@ -25,4 +25,17 @@ document.getElementById("sendButton").addEventListener("click", event => {
     event.preventDefault();
 });
 
-connection.start().catch(err => console.error(err.toString()));
\ No newline at end of file
+const start = async () => {
```

Comment:
```suggestion
async function start() {
```

Reply:
just curious why to switch this to the "function" syntax when the others are using the () => syntax? i'm okay with it but i'm more okay with consistency.

Reply:
This is a top-level function, rather than a lambda